src/veritysetup/veritysetup.c

```diff
@@ -81,6 +82,35 @@ static int run(int argc, char *argv[]) {
                         return 0;
                 }
 
+                if (argc > 7 && strcmp(argv[7], "-")) {
+                        for (;;) {
+                                _cleanup_free_ char *word = NULL;
+
+                                r = extract_first_word((const char **)&argv[7], &word, ",", EXTRACT_DONT_COALESCE_SEPARATORS | EXTRACT_UNESCAPE_SEPARATORS);
```

Comment:
uh, you are modifiying the argv[] array here, that's not OK, please do this in a temporary pointer you initialize from argv[7], but do not modify argv[7] itself.